recipe/meta.yaml

```diff
@@ -37,10 +37,6 @@ test:
     - if not exist "%LIBRARY_LIB%\\yaml.lib" exit 1   # [win]
     - if not exist "%LIBRARY_BIN%\\yaml.dll" exit 1   # [win]
 
```

Comment:
```suggestion

    # Check cmake and pkg-config scripts
    - test -f "${PREFIX}/cmake/yamlConfig.cmake"      # [not win]
    - test -f "${PREFIX}/lib/pkgconfig/yaml-0.1.pc"   # [not win]
```

Reply:
This also added the cmake test, which won't pass as it's not a cmake build anymore.

Reply:
Ah ok. Sorry let's drop that then. Thanks for catching this 🙂